README.rst

```diff
@@ -1250,6 +1246,77 @@ in the settings:
        }
    }
 
+Adding Checks and Alert settings from the device page
+-----------------------------------------------------
+
+We can add checks and define alert settings directly from the **device page**.
+
+To add a check, you just need to select an available **check type** as shown below:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/device-inline-check.gif
+  :align: center
+
+The following example shows how to use the
+`OPENWISP_MONITORING_METRICS setting <#openwisp_monitoring_metrics>`_
+to reconfigure the system for `iperf check <#iperf-1>`_ to send an alert if
+the measured **TCP bandwidth** has been less than **10 Mbit/s** for more than **2 days**.
+
+1. By default `Iperf checks <#iperf-1>`_ do not have any alert settings defined,
+but it is easy to create one through the device page as shown below:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/device-inline-alertsettings.gif
+  :align: center
```

Comment:
this GIF is 6 MB, can we find a way to be less heavy? Either optmize the GIF or use one or more screenshots?

Reply:
I'm unable to compress those GIFs to less than **5MB**, so I'm replacing them with screenshots.